person.rb

```diff
@@ -31,6 +32,3 @@ def validate_name(name)
     @corrector.correct_name(name)
   end
```

Comment:
- [ ] Please add rentals method in this class with `book` and `date` as parameters